zips/zip-0230.rst

```diff
@@ -315,23 +315,48 @@ An issuance action, ``IssueAction``, is the instance of issuing a specific Custo
 +-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
-|``noteSize * nNotes``        |``vNotes``                |``Note[nNotes]``                           |A sequence of note descriptions within the issuance action,          |
-|                             |                          |                                           |where ``noteSize`` is the size, in bytes, of a Note.                 |
+|``147 * nNotes``             |``vNotes``                |``IssueNote[nNotes]``                      |A sequence of note descriptions within the Issuance Action.          |
 +-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
 |``1``                        |``flagsIssuance``         |``byte``                                   |An 8-bit value representing a set of flags. Ordered from LSB to MSB: |
 |                             |                          |                                           | * :math:`\mathsf{finalize}`                                         |
 |                             |                          |                                           | * The remaining bits are set to :math:`0\!`.                        |
 +-----------------------------+--------------------------+-------------------------------------------+---------------------------------------------------------------------+
 
-The encodings of each of these elements are defined in ZIP 227 [#zip-0227]_.
+The encoding of ``IssueNote`` is described below.
+Note that we allow the number of notes (represented by ``nNotes``) to be zero. 
+This allows for issuers to create Issuance Actions to only finalize an issued Asset, without needing them to simultaneously issue more of that Asset. 
+
+Issue Note Description (``IssueNote``)
+--------------------------------------
+
+An issuance note, ``IssueNote`` contains the following fields:
+
++-----------------------------+--------------------------+--------------------------------------+--------------------------------------------------------------------+
+| Bytes                       | Name                     | Data Type                            | Description                                                        |
++=============================+==========================+======================================+====================================================================+
+|``43``                       |``recipient``             |``byte[43]``                          |The raw encoding of an Orchard Raw Payment Address, as              |
+|                             |                          |                                      |per protocol §5.6.4.2 ‘Orchard Raw Payment Addresses’.              |
```

Comment:
Currently the consensus protocol does not, strictly speaking, depend on this raw encoding. For example in note plaintexts only $\mathsf{d}$ is encoded. To preserve that property we should specify explicitly the encoding in terms of $\mathsf{d}$ and $\mathsf{pk_d}$.

Reply:
Updated in https://github.com/QED-it/zips/pull/75